app/models/build.rb

```diff
@@ -1,16 +1,61 @@
 class Build < ActiveRecord::Base
-  SHA1_REGEX = /\A[0-9a-f]{40}\Z/i
+  SHA1_REGEX = /\A[0-9a-f]{40,128}\Z/i
```

Comment:
Hmm, yeah. Probably a better idea to keep the SHA1 regex as is, and create a new regex to validate a Docker digest.